mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraMessageMapper.java

```diff
@@ -170,7 +170,14 @@ public Mono<MailboxCounters> getMailboxCountersReactive(Mailbox mailbox) {
                 }
                 return Mono.just(counters);
             })
-            .doOnNext(counters -> readRepair(mailbox, counters));
+            .doOnNext(counters -> {
+                Mono.fromRunnable(() -> {
+                    readRepair(mailbox, counters);
```

Comment:
Can we manage to subscribe here **only if read repairs are enabled** ie probability > 0 ?

Reply:
Sure, you mean something like:
```
.doOnNext(counters -> {
       if (probability > 0) {
              Mono.fromRunnable(() -> {
                   readRepair(mailbox, counters);
              })
             .subscribeOn(Schedulers.boundedElastic())
             .subscribe();
       }
});
```

Reply:
Yes

Reply:
@chibenwa I'm sorry could you guide how to access probability property here? 🤔

Reply:
from the configuration